web/core/templates/core/base.html

```diff
@@ -141,63 +107,30 @@
           </div>
         </div>
       </div>
-      <!-- mobile navigation controls -->
-      <div class="mobile-controls">
-        <span class="mobile-control-group mobile-header-icons">
-          <!-- Add more mobile controls here. These will be on the right side of the mobile page header section -->
-        </span>
-        <div class="mobile-control-group main-nav-icons">
-          <button id="nav-icon3"
-                  type="button"
-                  class="mobile-control toggle-menu"
-                  aria-expanded="false"
-                  aria-controls="navigation">
-            <span aria-hidden="true"></span>
-            <span aria-hidden="true"></span>
-            <span aria-hidden="true"></span>
-            <span aria-hidden="true"></span>
-            <span class="sr-only">Menu</span>
-          </button>
-        </div>
-      </div>
-      <div class="navigation-search full-width-nav container">
-        <div id="head-search" class="search-container featured-search">
-          <div class="d-block d-lg-none">
-            <a href="{% url 'core:index' %}"
-               class="drawer-logo-link first-level-link">
-              <img src="https://cdn.cdt.ca.gov/cdt/cagov/cagov-lockup/cagov-lockup-flag/cagov-lockup-flag-gradient/cagov-lockup-flag-gradient.svg"
-                   alt="CA.gov official logo"
-                   class="drawer-logo mb-4" />
-            </a>
-          </div>
-        </div>
-        <!-- navigation types -->
-        <nav id="navigation"
-             class="main-navigation dropdown"
-             aria-label="Main navigation">
-          <ul id="nav_list" class="top-level-nav">
-            <li class="nav-item">
-              <a class="first-level-link" href="{% url 'core:index' %}">Home</a>
-            </li>
-            <li class="nav-item">
-              <a class="first-level-link" href="{% url 'vital_records:index' %}">Vital records</a>
-            </li>
-          </ul>
-        </nav>
-      </div>
     </header>
-    <div id="main-content" class="main-content">
-      <div class="container">
-        <main class="main-primary">
-          <h1>
+    <main id="main-content" class="main-content">
+      <div class="main-header p-t-md bg-primary">
+        <div class="container">
+          <nav aria-label="You are here" class="breadcrumbs">
+            <ol>
+              <li>
+                <a href="#" title="Home page" class="no-underline underline-hover">< Go back to vital records home</a>
```

Comment:
Hmm, this feels like something that should be in a `{% block %}` that gets rendered in a template from the `vital_records` app and not the root `base.html` template?

Before it was a link _into_ the `vital_records` app. Now this is a link _out of_ this site and back to the "ca.gov/lafires" -- which is specific to the `vital_records` flow, right?